### ISSUE TITLE
3D Connexion support

### ISSUE THREAD

**Issue by PLAmenPRINT** [fstl-app/fstl#124]
Hello, I love the viewer very much, actually it is the only one stl viewer I am using from a long time -fast, simple and good looking.

The only thing that I would like to be improved is the navigation/ rotation.

Because there is no view block (as the one on fusion 360) it is difficult to orient the model on particular side.
I am using 3D Connexion all the time with Fusion 360, Bambu studio and others so it will be great if the FSTL have support it.

Thank You
Best Regards

P.S. I couldn't find the way to upload it as new feature.

**Comment by DeveloperPaul123** [comment]
Hello! Thanks for the issue. 

Could you expand a bit more on what you mean by navigation/rotation? Do you mean being able to rotate to specific views of the object (top, bottom, right, left, 3/4ths and so on)?

**Comment by PLAmenPRINT** [comment]
Hi, This is exactly what I mean - to set the object to particular view - front/ top, corner and so on.

Do You think the support of 3D Connexion mouse is possible implementation?

Thank You
Best Regards

**Comment by PLAmenPRINT** [comment]
Thanks a lot. I will download the last version.

**Comment by DeveloperPaul123** [comment]
Yes it may be on `master`. Maybe it's time for me to make a new release...

**Comment by PLAmenPRINT** [comment]
I will look forward to it :)

**Comment by DeveloperPaul123** [comment]
@PLAmenPRINT New version was just released. Sorry for the delay.

**Comment by PLAmenPRINT** [comment]
> [@PLAmenPRINT](https://github.com/PLAmenPRINT) No issues on my end. What error(s) are you seeing? Do you see anything in event viewer?

I am not facing any errors but after the installation the app does not launch at all.
Even if try to view some stl file it does not open it.
The behavior is like something from the app is missing.

I reinstall the previous version 0.10.0 and it works fine.

**Comment by DeveloperPaul123** [comment]
> > [@PLAmenPRINT](https://github.com/PLAmenPRINT?rgh-link-date=2025-01-23T19%3A12%3A17.000Z) No issues on my end. What error(s) are you seeing? Do you see anything in event viewer?
> 
> I am not facing any errors but after the installation the app does not launch at all. Even if try to view some stl file it does not open it. The behavior is like something from the app is missing.
> 
> I reinstall the previous version 0.10.0 and it works fine.

I can't reproduce the issue so I'm going to need more information to troubleshoot. If you can file a new issue with relevant information that would be helpful.
